text_extensions_for_pandas/array/test_tensor.py

```diff
@@ -55,6 +55,10 @@ def test_create(self):
         with self.assertRaises(TypeError):
             TensorArray(2112)
 
+        # Copy constructor
+        s_copy = TensorArray(s)
```

Comment:
Is there a need to use like a copy constructor? I thinks it's more common to explicitly copy like `TensorArray(s).copy()`

Reply:
Makes sense. Updating this PR accordingly.